test/mocha/mock.data.js

```diff
@@ -223,7 +223,7 @@ export const mockEntity2 = {
     // internalId is generated in tests.
     batchInvalidationCount: 0,
     openBatch: {},
-    minAssuranceForResolution: 2,
+    minAssuranceForResolution: 1,
```

Comment:
This is used in existing tests ... I'd rather us not change it but add another entity if we need another one.

Reply:
Added another mock entity [14ea658](https://github.com/digitalbazaar/bedrock-tokenization/pull/106/commits/14ea65865f175bbc56fc4dcf806a01e0e266ab13)